doc-surrealdb_versioned_docs/version-2.x/tutorials/define-a-schema.mdx

```diff
@@ -88,7 +88,7 @@ The above will return an error because the field must conform to `string::is::em
 
 ### Inserting fields that don’t exist in the schema
 
-In a schemafull table, since the fields need to be defined before you can populate them, If you add a field that doesn’t exist your data will be ignored. For example, in the user table you have only defined the `firstname` , `lastname` and `email` fields. If you introduce a `photoURI` field without defining the field in the `user` table it will return an error. 
+In a schemafull table, since the fields need to be defined before you can populate them, if you add a field that doesn’t exist, your data will be ignored. For example, in the user table, you have only defined the `firstname`, `lastname` and `email` fields. If you introduce a `photoURI` field without defining the field in the `user` table, it will not fail, but `photoURI` will be ignored as it is not a defined field.
```

Comment:
This is expected behaviour in 2.x. This sentence is premature but it is GA behaviour

Reply:
Thanks @Ekwuno! Reverted to "it will return an error." as it was before.